rotkehlchen/tests/utils/dataimport.py

```diff
@@ -1924,11 +1963,20 @@ def assert_binance_import_results(rotki: Rotkehlchen):
     assert trades == expected_trades
     assert asset_movements == expected_asset_movements
     assert expected_events == events
-    expected_warnings = [
-        '2 Binance rows have bad format. Check logs for details.',
-        'Skipped 4 rows during processing binance csv file. Check logs for details',
-    ]
-    assert warnings == expected_warnings
+    mocked_notifier = cast(MockRotkiNotifier, rotki.msg_aggregator.rotki_notifier)
+    assert mocked_notifier.pop_message() == MockedWsMessage(
```

Comment:
can you also assert that after poppping this message there is no other message left? To check we are not flooding it